inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -119,6 +119,12 @@
     'Instrumentation',
 ]
 
+GROBID_EXCEPTIONS = (
+    BadGatewayError, 
+    requests.exceptions.ConnectionError, 
+    requests.exceptions.HTTPError
+)
```

Comment:
shall we just [RequestException](https://requests.readthedocs.io/en/latest/api/#requests.RequestException)?